tools/test_utils.py

```diff
@@ -167,9 +167,7 @@ def run_tensorflow_pytests_from_artifacts(ngraph_tf_src_dir, tf_src_dir,
 
     # Check to see if we need to apply the patch for Grappler
     import ngraph_bridge
-    patch_file_name = "test/python/tensorflow/tf_unittest_ngraph" + (
-        "_with_grappler"
-        if ngraph_bridge.is_grappler_enabled() else "") + ".patch"
+    patch_file_name = "test/python/tensorflow/tf_unittest_ngraph.patch"
```

Comment:
The TF py tests will also run only with the graph pass?

Reply:
Yes, bridge C++ unit tests run with the optimizer, bridge Python and TF Python unit tests run without the optimizer. It seems a little excessive to test all the combinations here since the optimizer and non-optimizer paths are shared. Which configurations would we prefer testing?

Reply:
Which one is going to be the default/promoted more than the other, we should make sure all tests pass with that. If there isn't any then probably the above configuration is fine.

Reply:
The current default is non-optimizer. We could use this for all three but that'd leave the optimizer path untested.